mkdocs.yml

```diff
@@ -20,34 +20,7 @@ theme:
     code: Roboto Mono
 repo_url: https://github.com/neurobagel/documentation
 
-# Page Navigation
-nav:
-  - Welcome: "index.md"
-  - User guide:
-    - "user_guide/index.md"
-    - Running cohort queries:
-        - Search the public nodes: "user_guide/public_nodes.md"
-        - How to use the query tool: "user_guide/query_tool.md"
-        - Querying the API directly: "user_guide/api.md"
-    - Setting up your own Neurobagel node:
-        - Getting started: "user_guide/getting_started.md"
-        - Configuring a node: "user_guide/config.md"
-        - Maintaining a node: "user_guide/maintaining.md"
-    - Annotating your data:
-        - Preparing data for annotation: "user_guide/data_prep.md"
-        - Annotation tool guide: "user_guide/annotation_tool.md"
-        - Generating harmonized subject-level metadata: "user_guide/cli.md"
-  - Data models:
-    - Harmonized variables: "data_models/variables.md"
-    - Augmented BIDS data dictionaries: "data_models/dictionaries.md"
-    - Data files for Neurobagel graphs: "data_models/graph_data.md"
-    - Naming conventions for terms: "data_models/term_naming_standards.md"
-  - Contributing:
-    - How to contribute: "contributing/CONTRIBUTING.md"
-    - Our team: "contributing/team.md"
-  - Getting help: "getting_help.md"
-  - Glossary: "glossary.md"
-  - Citing Neurobagel: "cite.md"
+# For docs navigation definition, see docs/.nav.yml
```

Comment:
This is nice, but I would suggest you also put a short section in the README that says: do this when adding a new page